src/components/Icon/Icon.js

```diff
@@ -0,0 +1,188 @@
+                                  props.type === "all_on" ? "-227px -370px" :
+                                  props.type === "sun_on" ? "-310px -370px" :
+                                  props.type === "mon_on" ? "-391px -370px" :
+                                  props.type === "tue_on" ? "-474px -370px" :
+                                  props.type === "wed_on" ? "-557px -370px" :
+                                  props.type === "thu_on" ? "-638px -370px" :
+                                  props.type === "fri_on" ? "-720px -370px" :
+                                  props.type === "sat_on" ? "-803px -370px" :
+                                  props.type === "time_white" ? "-76px -488px" :
+                                  props.type === "time_black" ? "-158px -488px" :
+                                  props.type === "previous" ? "-227px -475px" :
+                                  props.type === "next" ? "-309px -475px" :
+                                  props.type === "minus" ? "-391px -475px" :
+                                  props.type === "plus" ? "-473px -475px" :
+                                  props.type === "add_white" ? "-73px -594px" :
+                                  props.type === "add_black" ? "-158px -594px" :
+                                  props.type === "check_circle" ? "-228px -582px" :
+                                  props.type === "play" ? "-309px -580px" :
+                                  props.type === "cancel_circle" ? "-391px -580px" :
+                                  props.type === "check_border" ? "-474px -580px" :
+                                  props.type === "check_square" ? "-550px -576px" : "0 0"};
+
+`;
+const cx = classNames.bind();
+
+export default function Icon(props) {
+  const { type, alt, style} = props;
+  let size = "xsm"
+  if(type === "drawer" || type === "check_off" || type === "check_on") {
+    size = "sm"
+  }
+  if(type === "all_off" || 
+     type === "sun_off" || 
+     type === "mon_off" || 
+     type === "tue_off" || 
+     type === "wed_off" || 
+     type === "thu_off" || 
+     type === "fri_off" || 
+     type === "sat_off" ||
+     type === "all_on" || 
+     type === "sun_on" || 
+     type === "mon_on" || 
+     type === "tue_on" || 
+     type === "wed_on" || 
+     type === "thu_on" || 
+     type === "fri_on" || 
+     type === "sat_on" ||
+     type === "next" ||
+     type === "minus" ||
+     type === "plus" ||
+     type === "previous" ||
+     type === "check_circle" ||
+     type === "play" ||
+     type === "cancel_circle" ||
+     type === "check_border" )  {
+       size = "md"
+     }
+  if(type === "check_square") {
+    size = "lg"
+  }
+
+  if (props.src) {
+    const { src } = props;
+    return (
+      <img
+        src={src}
+        alt={alt}
+        style={style}
+      />
+    );
+  }
+  return (
+    <I
+      style={style}
+      type={type}
+      size={size}
+      title={alt}
+    />
+  );
+}
```

Comment:
```suggestion
  return src ? (
    <img
      src={src}
      alt={alt}
      style={style}
    />
  ) : (
    <I
      style={style}
      type={type}
      size={size}
      title={alt}
    />
  );
}
```
삼항연산자를 활용하면 더 간결하고 좋을 것 같습니다!

Reply:
이부분도 src가 props에 없을 때도 있기때문에 if문으로 나눴던건데, 만약 props에 없다면 falsy한 값이기 때문에 삼항연산자에 잘 적용이 되는건가요??

Reply:
저도 실험해보겠습니다. 제 경험에 비추어보면 보통 props로 넘기지 않은 값은 undefined로 출력되니까 잘 실행될거라 예상되긴 합니다.

Reply:
적용됐습니다~